### PR TITLE
Created MyPromotions screen

### LINKED ISSUE #30: Set up "My Promos" page layout
## Task 🔨
Add the following features shown in the photo:
- Title for page
- Two dropdown buttons for filtering the list: options for the dropdown are still TBD
- "Show active deals" checkbox
- "Upload promo" button that opens the Upload Promotion page

You don't have to implement the cards for displaying the restaurant info, I'll probably be implementing them for the home page so you can use the same component once that's ready.

Screen Shot 2020-10-31 at 5 29 48 PM

### PR DESCRIPTION
**Pull request by richardyjtian**
Changed DropdownMenu into a template usable in Home and MyPromotions
Resolves #30

### CODE REVIEW

File: frontend/src/screens/MyPromotions.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+];
+
+const promotions: Promotion[] = [
+  {
+    title: "Happy Hour 2pm-4pm",
+    restaurantName: "Starbucks",
+    description: "Buy one, get one free on all Grande sized drinks!",
+    date: "Nov 11, 2020",
+    liked: false,
+    image: { src: "" },
```
Comment (ericrkuo) [inline frontend/src/screens/MyPromotions.tsx:44]: Hmm do you want some mock data that's accurate to what we have in the BE? It'll make it a bit easier for you so you won't need to make as many changes once the endpoint PR is pushed.
[SamplePromotionDate.txt](https://github.com/ubclaunchpad/foodies/files/5540276/SamplePromotionDate.txt)

Comment (ericrkuo) [reply frontend/src/screens/MyPromotions.tsx:44]: Let me know if you want any user sample data, but the BE endpoints for both promotions and users should be merged relatively soon

**Review by engineereng — APPROVED** [review]

File: frontend/src/screens/MyPromotions.tsx (hunk 1 above)
Comment (richardyjtian) [reply frontend/src/screens/MyPromotions.tsx:44]: Thanks for the sample data! Do you think we should be filtering and formatting the specific data we need (ie. for a PromotionCard) in the frontend or the backend?

File: frontend/src/types/Dropdown.ts [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+}
+
 export interface Dropdown {
   text: string;
-  options: DropdownOptions[];
+  options: DropdownOption[];
 }
 
-export interface DropdownOptions {
+export interface DropdownOption {
```
Comment (andrewzulaybar) [inline frontend/src/types/Dropdown.ts:10]: Ya I like this better, thanks for catching that 😛

**Review by andrewzulaybar — APPROVED** [review]
This looks great! 🚀  Feel free to merge when you're ready

**Pull request merged**